var/spack/repos/builtin/packages/memkind/package.py

```diff
@@ -38,6 +38,10 @@ class Memkind(AutotoolsPackage):
     depends_on('m4',       type='build')
     depends_on('numactl')
 
+    # memkind includes a copy of jemalloc.
+    # (It really shouldn't; with Spack, it should use an external jemalloc.)
```

Comment:
Can you rework this comment to point to this link: https://github.com/memkind/memkind#jemalloc ? Ideally I think we should patch `jemalloc` when building `memkind` but I have no idea of the amount of work that would require (so I'm fine merging this once we have the link in the comment)

Reply:
Done.